corehq/motech/generic_inbound/templates/generic_inbound/request_log_detail.html

```diff
@@ -158,4 +158,15 @@ <h3 class="panel-title">{% trans "Processing Attempts" %}</h3>
   </div>
 </div>
 
+<div class="form-actions">
+  {% if log.status != 'success' %}
```

Comment:
I suppose I could also check this on the back-end, but that doesn't seem necessary